taurus/demo/strehlow_and_cook.py

```diff
@@ -232,94 +336,99 @@ def make_strehlow_table(compounds):
         else:
             row.append(None)
 
-        x = list(filter(lambda y: y.name == band_msr.name, comp.measurements))
-        if x:
-            row.append(x[0].properties[0].value)
-        else:
-            row.append(None)
-
-        x = list(filter(lambda y: y.name == cryst_msr.name, comp.measurements))
-        if x:
-            row.append(x[0].properties[0].value)
-        else:
-            row.append(None)
-
-        x = list(filter(lambda y: y.name == color_msr.name, comp.measurements))
-        if x:
-            row.append(x[0].properties[0].value)
-        else:
-            row.append(None)
+        for term in terms:
+            x = list(filter(lambda y: y.name == term,
+                            comp.measurements[0].properties + comp.measurements[0].conditions))
+            if x:
+                row.append(x[0].value)
+            else:
+                row.append(None)
 
         output['content'].append(row)
 
     return output
 
 
-if __name__ == "__main__":
-    import pkg_resources
-    import json
-    import taurus.client.json_encoder as je
+def make_display_table(structured):
+    """Generate a Display Table from a passed Structured Table."""
```

Comment:
What exactly is a structure table? Is there a "structured table" class? This would be more readable with some type hints, or an expanded docstring.